config/site_settings.yml

```diff
@@ -876,11 +876,11 @@ posting:
   enable_system_message_replies:
     default: true
   personal_message_enabled_groups:
-    default: "3|11" # auto group trust_level_1
+    default: "11" # auto group trust_level_1
```

Comment:
I think we still keep admins & moderators in the defaults, no? Or does `SiteSetting.blah` also return the mandatory values regardless of defaults?

Reply:
I see it does, but IMO we should just keep it in both places, otherwise it raises the question "If it's mandatory why is it default too?"